.travis.yml

```diff
@@ -19,31 +18,22 @@ matrix:
   # tests pass on the stable versions of Go.
   fast_finish: true
   include:
-  - go: 1.9.x
-    install: dep ensure -v -vendor-only
-    script: go test -v -race -cover -timeout=1m ./...
-  - go: 1.10.x
-    install: dep ensure -v -vendor-only
-    script: go test -v -race -cover -timeout=1m ./...
-  - go: 1.11.x
-    before_install: skip
+  - go: 1.12.x
```

Comment:
Actually, as we no longer need to support `dep` we shouldn't have to specify a lot of the mattrix stuff per go version.  Probably this whole file could be simplified to:

```yaml
sudo: required

language: go

go:
  - 1.12
  - 1.13
  - tip

services:
  - docker

env:
  global:
    - VAULT_ADDR=http://127.0.0.1:8200
    - DEP_VERSION="0.5.0"
    - GO111MODULE=on

matrix:
  # Don't wait for tip tests to finish. Mark the test run green if the
  # tests pass on the stable versions of Go.
  fast_finish: true
  install:
    - go build ./...
    - go mod vendor
  script: go test -v -race -cover -timeout=1m -mod=vendor ./...

before_script:
  - docker run -d -p 2379:2379 quay.io/coreos/etcd /usr/local/bin/etcd -advertise-client-urls http://0.0.0.0:2379 -listen-client-urls http://0.0.0.0:2379
  - docker run -d -p 8500:8500 --name consul consul
  - docker run -d -p 8200:8200 --cap-add=IPC_LOCK -e 'VAULT_DEV_ROOT_TOKEN_ID=root' -e 'VAULT_DEV_LISTEN_ADDRESS=0.0.0.0:8200' vault:0.9.6
```

Reply:
👍